src/components/PipelineRun/RunDetails.tsx

```diff
@@ -63,66 +95,151 @@ export const RunDetails = ({ runId = "" }: RunDetailsProps) => {
-            <div className="text-sm text-gray-700 whitespace-pre-line">
-              {componentSpec.description}
-            </div>
+
+      {metadata && (
+        <div className="flex flex-col gap-2 text-xs text-secondary-foreground mb-2">
+          <div className="flex flex-wrap gap-x-6">
+            {metadata.id && (
+              <div>
+                <span className="font-semibold">Run Id:</span> {metadata.id}
+              </div>
+            )}
+            {metadata.root_execution_id && (
+              <div>
+                <span className="font-semibold">Execution Id:</span>{" "}
+                {metadata.root_execution_id}
+              </div>
+            )}
+          </div>
+          <div className="flex flex-wrap gap-6">
+            {metadata.created_by && (
+              <div>
+                <span className="font-semibold">Created by:</span>{" "}
+                {metadata.created_by}
+              </div>
+            )}
+          </div>
+          <div className="flex flex-wrap gap-x-6">
+            {metadata.created_at && (
+              <div>
+                <span className="font-semibold">Created at:</span>{" "}
+                {new Date(metadata.created_at).toLocaleString()}
+              </div>
+            )}
           </div>
-        )}
+        </div>
+      )}
+
+      {componentSpec.description && (
         <div>
-          <h3 className="text-md font-medium mb-1">Inputs</h3>
-          {componentSpec.inputs && componentSpec.inputs.length > 0 ? (
-            <ul className="list-disc list-inside text-sm text-gray-800">
-              {componentSpec.inputs.map((input) => (
-                <li key={input.name}>
-                  <span className="font-semibold">{input.name}</span>
-                  {input.type && (
-                    <span className="ml-2 text-gray-500">
-                      ({typeof input.type === "string" ? input.type : "object"})
-                    </span>
-                  )}
-                  {input.description && (
-                    <div className="text-xs text-gray-500 ml-4">
-                      {input.description}
-                    </div>
-                  )}
-                </li>
-              ))}
-            </ul>
-          ) : (
-            <div className="text-xs text-gray-400">No inputs</div>
-          )}
+          <h3 className="text-md font-medium mb-1">Description</h3>
+          <div className="text-sm text-gray-700 whitespace-pre-line">
+            {componentSpec.description}
+          </div>
         </div>
+      )}
+
+      <div>
+        <div className="flex items-center gap-2 mb-2">
+          <h3 className="text-md font-medium mb-1">Status: {status}</h3>
+          <div className="flex items-center gap-1 text-xs">
+            <span>{`(`}</span>
+            <StatusText statusCounts={statusCounts} />
+            <span>{`)`}</span>
+          </div>
+        </div>
```

Comment:
I think this entire section can be cleaned up. It doesn't have the `()`, but I think the cleaner code makes up for that.

```suggestion
        <div className="flex gap-2">
          <h3 className="text-md font-medium">Status: {status} </h3>
          <StatusText statusCounts={statusCounts} />
        </div>
```
<img width="300" alt="Screenshot 2025-06-26 at 1 32 31 PM" src="https://github.com/user-attachments/assets/08467ad5-962c-4b5c-88f4-b4354b4a221b" />

Reply:
Hmm. yeah this is fine without the brackets, so I'll simplify.